pkg/didmethod/httpuniversal/resolver.go

```diff
@@ -0,0 +1,116 @@
+/*
```

Comment:
we need timeout option and tlsconfig option similar to https://github.com/hyperledger/aries-framework-go/blob/master/pkg/didcomm/transport/http/outbound.go#L39

Reply:
Agree, used the configuration approach from `outbound.go`